desktop/src/main/java/bisq/desktop/primary/main/content/components/ChatMessagesComponent.java

```diff
@@ -235,7 +235,7 @@ private void onSendMessage(String text) {
 
         private void doSendMessage(String text) {
             ChatChannel<? extends ChatMessage> chatChannel = model.selectedChannel.get();
-            UserIdentity userIdentity = userIdentityService.getSelectedUserIdentity().get();
+            UserIdentity userIdentity = userIdentityService.getSelectedUserIdentity();
```

Comment:
Maybe good to add checkNotNull here as well to make it explicit that the value is expected to be not null.